scripts/lib/install

```diff
@@ -184,7 +184,7 @@ if ! apt-get install -y \
     "${ADDITIONAL_PACKAGES[@]}"; then
     set +x
     echo -e '\033[0;31m' >&2
-    echo "Installing packages failed; is network working and (on Ubuntu) the universe repository enabled?" >&2
+    echo "Installing packages failed; is network working?" >&2
```

Comment:
Is there a script that automatically enables universe on Ubuntu?

Reply:
I forgot the reason I made this commit. Or perhaps the real content of this commit was gone during a merge conflict resolution stage.